src/classes/pilot/Pilot.ts

```diff
@@ -1112,7 +1112,9 @@ class Pilot implements ICloudSyncable {
   }
 
   public get Counters(): ICounterData[] {
-    return this.features('Counters')
+    let counters:ICounterData[] = this.features('Counters')
+    if (this.ActiveMech) counters = counters.concat(this.ActiveMech.CountersOnlyMech)
+    return counters
```

Comment:
This counters was being iterated over which only grants counters on the pilot, regardless of mech.